README.md

```diff
@@ -52,6 +52,9 @@ receipt as returned by the iOS SDK (in which case it will be automatically base6
 Both productId and packageName (bundle ID) are optional, but when provided will be tested against.
 If the receipt does not match the provided values, an error will be returned.
 
+To verify auto-renewable subscriptions you need to provide `secret` field that contains your
+In-App Purchase Shared Secret.
```

Comment:
Sorry for the ignorance. But does Apple refer to this as "secret" or "password" in their documentation? I see their API takes a "password" field, but you call it a shared secret here. I don't mind this discrepancy if Apple caused that. But if they do have consistent naming, we should too. Would love your feedback on this.